modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -2703,6 +2719,22 @@ public void testUpdateSystemObjectDefinition() throws Exception {
 						ObjectFieldConstants.DB_TYPE_STRING,
 						RandomTestUtil.randomString(), StringUtil.randomId())));
 
+		// Enable localization must be true for modifiable object definitions
+
+		AssertUtils.assertFailure(
```

Comment:
@devpedrotavares why did you put this test inside the `testUpdateSystemObjectDefinition`?

Reply:
The method name `updateCustomObjectDefinition` is misleading. But we are updating objectDefinition1 which is a modifiable `system` definition.